.github/ISSUE_TEMPLATE/page-report.yml

```
@@ -0,0 +1,55 @@
+name: "[via MDN pages only]"
+description: Issue filed via "Report a problem with this content on GitHub" link on MDN pages.
+labels: ["via page report", "needs triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Before you start
+
+        **Want to edit this page?**
+        ↩ Go back and use the "Edit on GitHub" link on the page.
```

1. It would be better to be stronger - i.e. not "do you want to" but "please fix". Something like:

```suggestion
        **Can you make the edit yourself?**
        ↩ Go back and use the "Edit on GitHub" link on the page.
```

2. Can we make that "Edit link" automatic so instead it says "click here"?

---

@hamishwillee Ruth, Dipika, and I had a conversation about this today. We settled on next text to more strongly encourage folks to make edits, while still keeping with the overall tone. See https://github.com/mdn/content/pull/14555#discussion_r841877872

And no, unfortunately, issue templates don't have any dynamic elements. We can have some Markdown here, but it will be the same for _every_ issue. So we settled on this instruction as a next-best thing.